tests/integration/gm_gaia_health_test.go

```diff
@@ -86,28 +78,17 @@ func (s *DockerIntegrationTestSuite) TestAttesterSystem() {
 		}
 	}
 
-	// Extract validator key if found
-	var validatorArmoredKey string
-	if validatorKey != nil {
-		s.T().Logf("Extracting validator key...")
+	s.Require().NotNil(validatorKey, "validator key not found in keyring")
 
-		// Export private key in armor format (no passphrase for test keyring)
-		armoredPrivKey, err := kr.ExportPrivKeyArmor("validator", "")
-		require.NoError(s.T(), err, "failed to export validator private key")
-
-		s.T().Logf("Validator private key exported successfully (armor format)")
-		previewLen := 100
-		if len(armoredPrivKey) < previewLen {
-			previewLen = len(armoredPrivKey)
-		}
-		s.T().Logf("Armored key preview: %s...", armoredPrivKey[:previewLen])
-		validatorArmoredKey = armoredPrivKey
-	} else {
-		s.T().Log("No validator key found in keyring")
-	}
+	validatorArmoredKey, err := kr.ExportPrivKeyArmor("validator", "")
```

Comment:
validatorArmoredKey must be non empty, a later check verifies that, we can just remove everything around it and assert that here.